pymc/distributions/logprob.py

```diff
@@ -244,9 +245,16 @@ def joint_logp(
     # Only SimulatorRVs are allowed
     from pymc.distributions.simulator import SimulatorRV
 
+    temp_logp_var_dict_values = list(
+        filter(
+            lambda node: not isinstance(node.owner.op, Join),
+            temp_logp_var_dict.values(),
+        )
+    )
+
```

Comment:
Waiting for tests to run. Would this be an appropriate fix? Would anything else break (e.g. nodes not having a `.owner.op` attribute)?

Reply:
For now we can just overwrite it here and later remove it if the Aeppl one is changed.